apps/amiapp_flutter/ios/Runner/AppDelegate.swift

```diff
@@ -13,21 +13,13 @@ import FirebaseCore
     ) -> Bool {
         GeneratedPluginRegistrant.register(with: self)
         
-        FirebaseApp.configure()
```

Comment:
This line was deleted because FCM SDK will throw an exception at runtime saying that the FCM SDK is configured twice. 

flutter-fire must also configure it. So, this line is not needed.

Reply:
Do we plan to merge this to `main`? If yes, I suggest not to remove this and comment the code with notes for customers using our SDK without `flutter-fire`.

Reply:
@mrehan27 I just pushed a commit based on your suggestion. 

Is this what you were thinking?